PR: https://github.com/odoo/odoo/pull/178200

From: f10bd8ddcf99c9247dca417fefbc260454d74ea1
From: Gorash
Date: 2024-10-15 09:33:02

Breaking data model changes scores: del:4 + add:0, change matches:
-    _inherit = 'crm.lead'
-    _inherit = 'crm.team'
-    _inherit = 'res.users'
-    _inherit = 'sale.order'

Total Changes: 8

[IMP] *: adapt model class names to correspond to model names (apply script)

Part-of: odoo/odoo#178200
Related: odoo/enterprise#69762
Related: odoo/documentation#11115
Related: odoo/upgrade#6577
Related: odoo/design-themes#988
Related: odoo/upgrade-util#143

================================= pseudo patch: =================================

--- a/addons/sale_crm/models/crm_lead.py
+++ b/addons/sale_crm/models/crm_lead.py
@@ -8,7 +8,7 @@ from odoo.osv import expression
 
 
 class CrmLead(models.Model):
-    _inherit = 'crm.lead'
+    _inherit = ['crm.lead']
 
     sale_amount_total = fields.Monetary(compute='_compute_sale_data', string="Sum of Orders", help="Untaxed Total of Confirmed Orders", currency_field='company_currency')
     quotation_count = fields.Integer(compute='_compute_sale_data', string="Number of Quotations")

--- a/addons/sale_crm/models/crm_team.py
+++ b/addons/sale_crm/models/crm_team.py
@@ -6,7 +6,7 @@ from odoo.tools import SQL
 
 
 class CrmTeam(models.Model):
-    _inherit = 'crm.team'
+    _inherit = ['crm.team']
 
     def _compute_dashboard_button_name(self):
         super(CrmTeam, self)._compute_dashboard_button_name()

--- a/addons/sale_crm/models/res_users.py
+++ b/addons/sale_crm/models/res_users.py
@@ -5,6 +5,6 @@ from odoo import fields, models
 
 
 class ResUsers(models.Model):
-    _inherit = 'res.users'
+    _inherit = ['res.users']
 
     target_sales_invoiced = fields.Integer('Invoiced in Sales Orders Target')

--- a/addons/sale_crm/models/sale_order.py
+++ b/addons/sale_crm/models/sale_order.py
@@ -5,7 +5,7 @@ from odoo import fields, models
 
 
 class SaleOrder(models.Model):
-    _inherit = 'sale.order'
+    _inherit = ['sale.order']
 
     opportunity_id = fields.Many2one(
         'crm.lead', string='Opportunity', check_company=True,
